### PR TITLE
Updates changelog after 3.2.1 release

### PR DESCRIPTION
**Pull request by ashfurrow**
Followup from #2352.

### CODE REVIEW

File: CHANGELOG.yml [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+    version: 3.2.2
+    details: Scope not known yet
+    dev:
+    user_facing:
+      - Adds timer for artwork view that refreshes UI when the associated live sale opens - ash
+
+releases:
+  - version: 3.2.1
     details: Scope not know yet
+    date: May 22, 2017
```
Comment (orta) [inline CHANGELOG.yml]: nice one

Comment (ashfurrow) [inline CHANGELOG.yml]: Not my first rodeo :)

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">You need to set the App’s plist version to 3.2.2</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ashfurrow** [comment]
Awww snap. Thanks Danger.

**Pull request merged**